client/components/fixed-navigation-header/index.tsx

```diff
@@ -4,7 +4,7 @@ import Breadcrumb, { Item as TBreadcrumbItem } from 'calypso/components/breadcru
 
 const Header = styled.header`
 	position: fixed;
-	z-index: 1;
+	z-index: 10;
```

Comment:
Core's button components have z-index of 1, moving our header components to 10 and 20 to keep them on top while scrolling.